framework/src/main/java/org/motechproject/uitest/page/DataServicesPage.java

```diff
@@ -16,8 +19,15 @@
     public static final By DATA_SERVICES_BUTTON = By.id("modulelink_data-services");
     public static final By SCHEMA_EDITOR_BUTTON = By.id("mdsTab_schemaEditor");
     public static final By BROWSE_INSTANCES_BUTTON = By.id("browseInstancesButton");
-    public static final By ADD_NEW_INSTANCE_BUTTON = By.id("addNewInstanceButton");
     public static final By ENTITY_SPAN = By.id("select2-chosen-2");
+    public static final By FIELD_TYPE_DROPDOWN = By.id("new-field-type");
+    public static final By FIELD_DISPLAY_NAME = By.xpath("//div[@id='new-field-displayName']/input");
+    public static final By FIELD_NAME = By.xpath("//div[@id='new-field-name']/input");
+    public static final By FIELD_TYPE_BOOLEAN = By.xpath("//div[@class='select2-result-label']/div/div/strong[text() = 'Boolean']");
```

Comment:
we should avoid to specific xpath expressions - function names or css classes can change. We can add ids to these elements in Motech if they are missing.